src/modules/classlist.js

```diff
@@ -1,23 +1,17 @@
 export default class ScoresList {
-    allscores = [];
-
-    savescore() {
-      const sco = JSON.stringify(this.allscores);
-      localStorage.setItem('scores', sco);
-    }
-
-    addscore(names, scorevalue) {
-      const scoreArr = {
-        names,
-        scorevalue,
-      };
-      this.allscores.push(scoreArr);
-      this.savescore();
-    }
-
-    getStoredscore() {
-      this.allscores = JSON.parse(
-        localStorage.getItem('scores'),
-      );
-    }
+  async addscore(names, scorevalue) {
+    this.a = 'hi';
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/odai/scores', {
+      method: 'POST',
+      body: JSON.stringify({
+        user: names,
+        score: scorevalue,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .catch((error) => error('Error:', error));
+  }
```

Comment:
- [ ] Nice job working with your async javascript and consuming the API! 💯 However, according to the requirements, I would suggest you use async and await instead of `.then` and `.catch` chains, this will improve the readability and organization of your code, please consider fixing this! 🤓 🛠️ 

Requirements:
![image](https://user-images.githubusercontent.com/77069011/203558410-149804c6-8f1c-4526-93a6-27e4d2e68300.png)